DESIGN.md

```diff
@@ -294,7 +294,7 @@ connected Envoy proxies with a list of clusters, mapping of service name to list
 The `ListEndpointsForService` method will be provided by the OSM component, which we refer to
  as the **Mesh Catalog** in this document.
 
-The Mesh Catalog will have access to the `MeshSpec`, `CertificateManager`, and the list of `EndpointsProvider`s.
+The Mesh Catalog will have access to the `MeshSpec`, `CertificateManager`, and the list of `EndpointsProvider`s and `ServiceProvider`s.
```

Comment:
let's not add the Service Provider's now, once all the PR's and changes related to service providers is checked in, lets add and update the Design.md to contain the needed information.

There will be a section on Service Providers too just like Endpoint Providers